.travis.yml

```diff
@@ -3,8 +3,6 @@ language: python
 python:
   - 2.6
```

Comment:
And actually, let's remove 2.6 support too. 2.6 is end of life as well. We may support 2.6 or 3.3 by happenstance, but it's not something I want to strive to maintain.

Reply:
Additionally, let's remove `setup.py:80,83-84` to indicate 2.6 and 3.2-3.3 removal.